src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -469,13 +465,15 @@ def _write_genome_json(self):
 def main() -> None:
     """Main script entry-point."""
     parser = ArgumentParser(description="Parse a GenBank file and create cleaned up files from it.")
-    parser.add_argument_src_path("--gb_file", required=True, help="Sequence accession file")
-    parser.add_argument("--prefix", required=True, help="Prefix to add to every feature ID")
-    parser.add_argument("--prod_name", required=True, help="Production name for the species")
+    parser.add_argument_src_path("--gb_file", required=True, help="sequence accession file")
+    parser.add_argument("--prefix", required=True, help="prefix to add to every feature ID")
+    parser.add_argument("--prod_name", required=True, help="production name for the species")
     parser.add_argument_dst_path(
-        "--out_dir", default=Path.cwd(), help="Output folder where the generated files will be stored"
+        "--out_dir", default=Path.cwd(), help="output folder where the generated files will be stored"
     )
+    parser.add_log_arguments()
     args = parser.parse_args()
+    init_logging(args.log_level)
```

Comment:
do we need to pass `args.log_file`, `args.log_file_level` as well? Thanks

Reply:
Those arguments are not available with `parser.add_log_arguments()` (without setting the flag of this method to `True`).